### DISCUSSION TITLE
06310a78-e9e6-44af-9af2-c2973de6d9ce

### DISCUSSION THREAD

**Discussion by giscus** [varofla/blog_giscus#19] [General]
# 06310a78-e9e6-44af-9af2-c2973de6d9ce

공유기에 20만원을 태워?! 공유기를 위한 N100 기반 미니PC를 구매했습니다.

https://www.varofla.com/06310a78-e9e6-44af-9af2-c2973de6d9ce

**Comment by artman000** [comment]
안녕하세요 차완기님, 
PCB제작 및 3D제품 제작 관련해서 정보를 찾아보다가 네이버 블로그를 통해 들어오게 되었습니다.
개인적으로 대화를 드리고 싶은데, 다른 루트를 찾지못해 이렇게 댓글로 문의드립니다! 
프로젝트를 진행중인데 차완기님께 문의를 드리고 싶은데 방법이 있을까요?   
정말 많은 열정과 꼼꼼함이 모든 포스팅에서 보여서 너무 좋았습니다.

**Reply by varofla** [reply]
안녕하세요, @artman000 님

좋게 봐주셔서 감사합니다 :)
아래 이메일로 남겨주시면 최대한 도움 드리겠습니다.
<max.cha@varofla.com>
